components/resources/quizzes/careerQuiz/career-quiz.tsx

```diff
@@ -50,7 +55,7 @@ const CareerQuiz = () => {
         };
       }),
       currentQuestion: 0,
-      progress: 0,
+      progress: quizData.questions.length,
```

Comment:
I'm pretty sure this useEffect is not needed, and you can just use this object in the state initializer at line 38